### ISSUE TITLE
How to train your network?

### ISSUE THREAD

**Issue by zhanjiahui** [BERYLSHEEP/AdvStyle#2]
Your paper is pretty good !!
 
1. In your paper, I note that some components are fixed and some components are trainable. However, I haven't found any code in your project which represents training some components. Could you please tell me how to train it?
2. I know that thelta is an important trainable parameter, but I am fused about if alpha is trainable?

**Comment by Johnson-yue** [comment]
+1

**Comment by zxwedu** [comment]
+1

**Comment by yeohhoo** [comment]
Hello, for your Resnet, is it training in a way of Siamese network? Because , i found that you calculate the similarity score of two images in your paper. That seems like Siamese network. I can't found any training code. Can you tell me how to train? Thanks

**Comment by yeohhoo** [comment]
Maybe i am wrong. I can't understand the usage of Resnet18. In 3.2.2, you used the cross-entropy. In 3.3.2，you calculated the identity loss by the cosine distance function. Firstly, training the network in classifier. Then, Extracting the features of images using the classifier. Is this right? Could you tell me which one is right? Thank you very much.
